src/base_parsers.rs

```diff
@@ -40,6 +40,7 @@ where
     many1(alpha_num().or(token('_')))
 }
 
+#[allow(dead_code)]
```

Comment:
Is this not being used anywhere?

Reply:
Not in the current code base:
```
warning: function `words` is never used
  --> src/base_parsers.rs:43:15
   |
43 | pub(crate) fn words<I>() -> impl Parser<I, Output = String>
   |               ^^^^^
   |
   = note: `#[warn(dead_code)]` on by default

warning: `lustre_collector` (lib) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 2.87s
```
But I haven't checked other PRs.

Reply:
Let's remove it. I don't want to get into the habit of using linting rules to escape clippy.